### PR TITLE
Focus terminal for shell tasks.

### PR DESCRIPTION
**Pull request by yuchenshi**
### Description

Makes it easier to answer prompts and harder to hit the `init` button twice.

### Scenarios Tested

Not tested :(

### Sample Commands

Click the `firebase init`, deploy, and start emulator buttons in left panel.

### CODE REVIEW

**Review by joehan — APPROVED** [review]

**Comment by yuchenshi** [comment]
Marking as draft since this is untested

File: firebase-vscode/src/data-connect/terminal.ts [hunk 1]
```diff
@@ -54,15 +54,15 @@ export function runTerminalTask(
...
-        new vscode.ShellExecution(command, executionOptions),
-      ),
+    const task = new vscode.Task(
+      { type },
+      vscode.TaskScope.Workspace,
+      taskName,
+      "firebase",
+      new vscode.ShellExecution(command, executionOptions),
     );
+    task.presentationOptions = { focus: true };
```
Comment (yuchenshi) [inline firebase-vscode/src/data-connect/terminal.ts:64]: On a second thought, maybe we don't need focus for running the emulators? That command almost never prompts for anything interactively. Focusing the terminal for init and deploy makes sense though

Comment (hlshen) [reply firebase-vscode/src/data-connect/terminal.ts:64]: I agree, we can add it to the call signature

Comment (yuchenshi) [reply firebase-vscode/src/data-connect/terminal.ts]: Done

**Pull request merged**